frontend/src/components/tracks/index-pm.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import Header from './header/index';
+import Footer from '../shared/footer/index';
+import TrackImage from './track-image/index';
+import WhatYouLearn from './what-you-learn/index';
+import OfferResponsibilities from './offer-responsibilities/index';
+import Expectations from './expectations/index';
+import { or, header, expectations } from './data/pm';
+
+/**
+ * The Apply Now button is not abstracted away in its own component, because it's temporary there
+ * TrackImage is for now the same for all tracks. It can be later made more flexible if track heads
+ * decide to use different images.
+ */
+class TrackPM extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <Header data={header} logo="/assets/tracks/dps-logo-white.png" />
+        <TrackImage />
+        <OfferResponsibilities data={or} />
+        <Expectations data={expectations} />
+        <WhatYouLearn />
+        <div
+          className="u-content"
+          style={{
+            display: "flex",
+            textAlign: "center",
+            justifyContent: "center",
+            marginTop: "200px",
+          }}
+        >
+          <a 
+            className="u-button u-button__apply u-button--reversed" 
+            href="https://utum.typeform.com/to/qkkFHF?ref=website&entry-point=track-page"
```

Comment:
This link should contain the selected track, shouldn't it?

Reply:
depends on this: https://github.com/DigitalProductschool/website/pull/50#discussion_r248213584 :)